downstream/assemblies/platform/assembly-gs-key-functionality.adoc

```diff
@@ -13,9 +13,12 @@ endif::[]
 
 = Key functionality and concepts
 
-With {PlatformNameShort}, you can create, manage, and scale automation for your organization across users, teams, and regions. 
+With {PlatformNameShort}, you can create, manage, and scale automation for your organization across users, teams, and regions. See the following functionality and concepts of {PlatformNameShort} for more details. 
 
-See the following functionality and concepts of {PlatformNameShort} for more details. 
+The release of {PlatformNameShort} 2.5 introduces an updated, unified user interface (UI) that allows you to interact with and manage each component part of the platform.
```

Comment:
"each component part" seems redundant. I suggest using one or the other.

Reply:
updated